client/webpack.config.js

```diff
@@ -30,6 +31,21 @@ module.exports = {
           'css-loader'
```

Comment:
could be useful to use some kind of postcss-loader optimization. Not a requirement for right now but it could be useful for prod at least in the future.

Reply:
Certainly! I'd love for us to set that up in a future improvement task